src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -23,6 +23,7 @@
 // IWYU pragma: no_forward_declare Tensor
 
 namespace GeneralizedHarmonic {
+namespace gauges {
```

Comment:
Is this commit identical to the one in #2225 ?

Reply:
Whoops yes, you're right. Sorry about this! I don't think it's needed in this one, but 2225 is also close to done :)